<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarBlock</class>
 <widget class="QWidget" name="CarBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color: gray
}

QMessageBox QPushButton:hover {
border-radius: 5px;
background: rgb(255,140,0);
}

QMessageBox QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(255,140,0), stop: 0.7 rgb(255,105,0));
min-height: 30px;
min-width: 70px;
}

QMessageBox QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(255,105,0);
}
</string>
  </property>
  <widget class="QLabel" name="lblPhoto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/car.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReserve">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(255,140,0);
}

QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(255,140,0), stop: 0.7 rgb(255,105,0));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(255,105,0);
}
</string>
   </property>
   <property name="text">
    <string>Rezerwuj</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Status samochodu</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/rented.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeedImage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>95</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Przebieg</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/speed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlateImage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Numer rejestracyjny</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/plate.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>156</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/delete.png</normaloff>:/images/images/delete.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblInsuranceImage">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Koniec ubezpieczenia</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/insurance.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblInspectionImage">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>95</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Koniec przeglÄ…du</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/inspection.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddInsurance">
   <property name="geometry">
    <rect>
     <x>673</x>
     <y>52</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/savetodb.png</normaloff>:/images/images/savetodb.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddInspection">
   <property name="geometry">
    <rect>
     <x>673</x>
     <y>97</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/savetodb.png</normaloff>:/images/images/savetodb.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditInsurance">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>52</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 5px;
color:gray;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditInspection">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>97</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 5px;
color:gray;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnViewNotes">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Uwagi</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/comments.png</normaloff>:/images/images/comments.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lblLicensePlate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>55</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
color: gray;
background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Nr rejestracyjny</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lblMileage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
color: gray;
background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Przebieg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lblCarName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>650</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
color: gray;
background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Marka Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string>Dodaj </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/image.png</normaloff>:/images/images/image.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddMileage">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>97</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/savetodb.png</normaloff>:/images/images/savetodb.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddLicensePlate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>52</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/savetodb.png</normaloff>:/images/images/savetodb.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblPersonImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>WypoÅ¼yczony przez ...</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/person.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPerson">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>195</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnViewRepairs">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Naprawy</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/tools.png</normaloff>:/images/images/tools.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIsVisible">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>37</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>WidocznoÅ›Ä‡ samochodu</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/visible.png</normaloff>:/images/images/visible.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPDF">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Raport PDF</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/report.png</normaloff>:/images/images/report.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOil">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Olej</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(210,210,210);
}

QPushButton{
color: rgb(70,70,70);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(230,230,230), stop: 0.7 rgb(210,210,210));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(230,230,230);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/oil.png</normaloff>:/images/images/oil.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
